Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -56,3 +56,28 @@
 DigiToHybridRawRepack = cms.Sequence( DigiToHybridRawRepackTask )
 DigiToVirginRawRepack = cms.Sequence( DigiToVirginRawRepackTask )
 DigiToSplitRawRepack = cms.Sequence( DigiToRawRepackTask, DigiToVirginRawRepackTask )
+
+from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
+siStripDigisHLT = siStripDigis.clone(ProductLabel = "rawDataRepacker")
+
+from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppression
+siStripZeroSuppressionHLT = siStripZeroSuppression.clone(
+    RawDigiProducersList =cms.VInputTag(cms.InputTag("siStripDigisHLT","VirginRaw"), cms.InputTag("siStripDigisHLT","ProcessedRaw"), cms.InputTag("siStripDigisHLT","ScopeMode")
+    ))
+    
+from RecoLocalTracker.SiStripClusterizer.DefaultClusterizer_cff import *
+siStripClustersHLT = cms.EDProducer("SiStripClusterizer",
+                                    Clusterizer = DefaultClusterizer,
+                                    DigiProducersList = cms.VInputTag(
+                                        cms.InputTag('siStripDigisHLT','ZeroSuppressed'),
+                                        cms.InputTag('siStripZeroSuppressionHLT','VirginRaw'),
+                                        cms.InputTag('siStripZeroSuppressionHLT','ProcessedRaw'),
+                                        cms.InputTag('siStripZeroSuppressionHLT','ScopeMode')),
+                                )
+
+from RecoLocalTracker.SiStripClusterizer.SiStripClusters2ApproxClusters_cfi import *
+from Configuration.ProcessModifiers.approxSiStripClusters_cff import approxSiStripClusters
+approxSiStripClusters.toModify(SiStripClusters2ApproxClusters, inputClusters = "siStripClustersHLT")
```

Comment:
You are modifying here the original `SiStripClusters2ApproxClusters`: the correct procedure could be to clone a `SiStripClusters2ApproxClustersHLT` instead, or apply the modifier in SiStripClusters2ApproxClusters_cfi (or better, in a SiStripClusters2ApproxClusters_cff.py that imports it)